packages/authentication/integration-tests/utils/get-init-module-config.ts

```diff
@@ -0,0 +1,33 @@
+import { Modules, ModulesDefinition } from "@medusajs/modules-sdk"
+
+import { DB_URL } from "./database"
+
+export function getInitModuleConfig() {
+  const authenticationModuleOptions = {
```

Comment:
```suggestion
  const moduleOptions = {
```
to facilitate the copy/paste 😆

Reply:
you got me 😂